data.js

```diff
@@ -0,0 +1,36 @@
+const nameInput = document.getElementById('username');
+const emailInput = document.getElementById('email');
+const textAreaBoxInput = document.getElementById('msg');
+const formStorage = document.getElementById('form');
+
+let userInfo = JSON.parse(localStorage.getItem('userInfo'));
+
+if (userInfo !== null) {
+  if (userInfo.hasOwnProperty.call('username')) {
+    nameInput.value = userInfo.username;
+  }
+  if (userInfo.hasOwnProperty.call('email')) {
+    emailInput.value = userInfo.email;
+  }
+  if (userInfo.hasOwnProperty.call('msg')) {
+    textAreaBoxInput.value = userInfo.msg;
+  }
```

Comment:
- [ ] The data is being stored correctly to local storage, but after reloading the page, the `userInfo.hasOwnProperty.call('...')` is returning false even when there is data in the `userInfo` object, and this is preventing the page to load the data in the form. Kindly fix this bug and use another conditional to check if there is data in those data fields so they are correctly populated after the page reloads.